### PR TITLE
Large deployments guide includes mention to proxy (bsc#1199577)

### PR DESCRIPTION
**Pull request by rjmateus**
Signed-off-by: Ricardo Mateus <rmateus@suse.com>

# Description

Add mention to proxy on large scale deployments

Implementation of: https://github.com/SUSE/spacewalk/issues/17856

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.3
- [X] Manager-4.2
- [X] Manager-4.1

# Links

Fixes https://github.com/SUSE/spacewalk/issues/17856

### CODE REVIEW

File: modules/specialized-guides/pages/large-deployments/overview.adoc [hunk 1]
```diff
@@ -21,6 +21,9 @@ There are two main ways to manage large scale deployments.
 You can manage them with a single {productname} Server, or you can use multiple servers in a hub.
 Both methods are described in this book.
 
+With large scale environments one should also consider the usage of {productname} Proxies, for more information see xref:installation-and-upgrade:install-proxy.adoc[].
+Sizing and location of the Proxies will dependent on the deployment topology.
```
Comment (keichwa) [inline modules/specialized-guides/pages/large-deployments/overview.adoc:25]:
```suggestion
With large scale environments one should also consider the usage of {productname} Proxies.
Sizing and location of the proxies will dependent on the deployment topology.
For more information about {productname} proxies, see xref:installation-and-upgrade:install-proxy.adoc[].
```
Arguable, but in general we prefer more-info-refs at the end of text chunks.

Thanks for the PR!

**Review by keichwa — APPROVED** [review]

**Comment by rjmateus** [comment]
@keichwa I resolved the conflicts and re-organized the docs to have the link at the end

**Review by keichwa — APPROVED** [review]
Thank you, @rjmateus !

**Review by 0rnela — APPROVED** [review]

**Comment by keichwa** [comment]
@rjmateus I assume that you do not have merge rights thus we can merge it soon.

**Comment by rjmateus** [comment]
@keichwa you are right, I don't have merge rights on this repo.

**Pull request merged**